### ISSUE TITLE
I tested your ResNeXt-101 pretrained model in UCF-101 and the accuracy is only 87.7%

### ISSUE THREAD

**Issue by dontLoveBugs** [kenshohara/3D-ResNets-PyTorch#121]
Hi, I test your ResNeXt-101 fine-tuned on UCF-101 (split1), and use the 3783 test videos. The transforms codes is as follows(I think it is the same with your test code): 

```

    spatial_transform = Compose([
        Scale(int(args.sample_size / args.scale_in_test)),
        CornerCrop(args.sample_size, args.crop_position_in_test),
        ToTensor(args.norm_value), norm_method
    ])
    temporal_transform = TemporalCenterCrop(args.sample_duration)
    target_transform = ClassLabel()
    validation_data = get_validation_set(
        args, spatial_transform, temporal_transform, target_transform)
    val_loader = torch.utils.data.DataLoader(
        validation_data,
        batch_size=1,
        shuffle=False,
        num_workers=args.n_threads,
        pin_memory=True)

```

And the test results is:
![image](https://user-images.githubusercontent.com/17598076/50383833-5dfb9100-06f6-11e9-88b9-a876622d0e34.png)
I dont know why and I hope you can reply to me.

**Comment by xiaoiker** [comment]
Hello, can I ask you a question. After we get the pretrained model, say resnext-101-kinetics-ucf101_split1.pth, do we still need to finetuning if we want to get the result report? I test it directly, butI get the accuracy about 0%.
